packages/types/src/product/common.ts

```diff
@@ -9,10 +12,43 @@ export enum ProductStatus {
 }
 
 /**
- * DTO in and out of the module (module API)
+ * @interface
+ * 
+ * A product's data.
+ * 
+ * @privateRemarks
+ * TODO: This DTO should represent the product, when used in config we should use Partial<ProductDTO>, it means that some props like handle should be updated to not be optional
+ * 
```

Comment:
@adrien2p this seems to be your comment based on git blame, do you know what this means?

Reply:
@shahednasser adrien mentions that this can be removed. Its stale now.